src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioMetadataReferenceManager.MetadataCache.cs

```diff
@@ -13,15 +13,10 @@ internal sealed partial class VisualStudioMetadataReferenceManager
     {
         private sealed class MetadataCache
         {
-            private const int InitialCapacity = 64;
-            private const int CapacityMultiplier = 2;
```

Comment:
these values were used to cleanup the dictionary if the assemblies were GCed.  But since they're now strongly held, that cleanup is meaningless, so it goes away.  Note that actual testing over users showed no impactful difference in memory held with this feature on/off, indicating in practice that people do not run into significant scenarios where they bring these assemblies into memory, but then end up not needing them.